overlord/devicestate/devicemgr.go

```diff
@@ -435,6 +435,26 @@ func (m *DeviceManager) ensureOperational() error {
 	return nil
 }
 
+var startTime time.Time
+
+func init() {
+	startTime = time.Now()
```

Comment:
I think this should be set in in DeviceManager when devicestate.Manager() is called.

Reply:
all the devicemgr bits here are actually from the other PR, https://github.com/snapcore/snapd/pull/9035, so I will address this feedback there

Reply:
This was my original implementation. I actually measured and on my machine there's a 120ms between the init and Manager. So I think I would prefer to keep it this way, it's messier but gives a slightly better number.